proposals/v3/reads.md

```diff
@@ -0,0 +1,157 @@
+# Synse Reads
```

Comment:
There is no direct read from synse server -- the plugin is constantly reading from devices on a loop and updating internal state. When synse reads from a plugin, it reads from that internal state, not directly from the device.

readcached basically just gets a dump of *all* the internal read state (and possibly older cached state if it is enabled for the plugin) instead of read state for just one device.

Reply:
Thanks